doc/index.rst

```diff
@@ -1,110 +0,0 @@
-.. _ros2_control_demos:
```

Comment:
Should we move the info regarding the examples from README.md into this file to keep control.ros.org working?
What about the figures in the doc folder, they are at least not up-to-date?